.storybook/preview.tsx

```diff
@@ -43,24 +39,22 @@ const globalStyles = css`
   ${baseStyle}
 `;
 
-export const decorators = isEmbeddingSDK
-  ? [] // No decorators for Embedding SDK stories, as we want to simulate real use cases
-  : [
-      renderStory => {
-        if (!document.body.classList.contains("mb-wrapper")) {
-          document.body.classList.add("mb-wrapper");
-        }
-        return (
-          <EmotionCacheProvider>
-            <ThemeProvider>
-              <Global styles={globalStyles} />
-              <CssVariables />
-              {renderStory()}
-            </ThemeProvider>
-          </EmotionCacheProvider>
-        );
-      },
-    ];
+export const decorators = [
+  renderStory => {
+    if (!document.body.classList.contains("mb-wrapper")) {
+      document.body.classList.add("mb-wrapper");
+    }
```

Comment:
Wouldn't `PublicComponentStylesWrapper` already wrap the component with `.mb-wrapper`? Is this because some stories/components are not wrapped with a style wrapper?

Reply:
This is for the core app storybook, in the core app we added that class to the body: https://github.com/metabase/metabase/blob/2aac6c411c9ea4d448cfc8bf398b6dcd995c2e39/resources/frontend_client/index_template.html#L47

Reply:
Oh, I misread the file name - all good then!